Database/Entities/Faction.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Database.Entities;
+
+/// <summary>
+/// Represents an in-game faction.
+/// </summary>
+public partial class Faction
+{
+    /// <summary>
+    /// Primary key of this record in the database.
+    /// </summary>
+    public int ID { get; }
+
+    /// <summary>
+    /// How the faction is referred to in the game.
+    /// </summary>
+    public string Name { get; set; }
+
+    /// <summary>
+    /// A collection of <c>Creature</c> entities that belong to this faction. <para/>
+    /// This collection is initialized as an empty list by default.
```

Comment:
I think line 10 is redundant, my reasoning being similar to the `System.Drawing` of earlier versions.
Also, for other cases, `<para/>` should be on a new line. I.e. at the start of the paragraph it is declaring.

Reply:
Still pending.